contracts/src/v0.8/dev/automation/2_1/KeeperRegistry2_1.sol

```diff
@@ -546,15 +546,13 @@ contract KeeperRegistry2_1 is KeeperRegistryBase2_1, OCR2Abstract, Chainable, ER
    * transmitter and the exact gas required by the Upkeep
    */
   function _performUpkeep(
-    Trigger triggerType,
     AutomationForwarder forwarder,
     uint256 executeGas,
+    bytes4 selector,
```

Comment:
This seems like a separate full feature for customers to select their perform function and it would be good to split this out into a separate discussion

Reply:
FWIW we can still do that even with these changes. These changes give us the _option_ to pursue this solution but it doesn't mean we have to do it this we if we thing of something better later. It would be perfectly acceptable imho to have this configuration but never officially support it.